packages/wonder-blocks-layout/src/components/__tests__/media-layout.test.tsx

```diff
@@ -78,185 +76,177 @@ describe("MediaLayout", () => {
             });
 
             // Assert
-            // @ts-expect-error [FEI-5019] - TS2571 - Object is of type 'unknown'.
             expect(args.mediaSize).toEqual("large");
         });
     });
 
-    describe("styleSheets", () => {
-        const testSizes = {
-            small: [640, 480],
-            medium: [800, 600],
-            large: [1200, 800],
-        } as const;
```

Comment:
The size arrays were not being used. I changed this to a `describe.each` of the media sizes instead of this data type and the `for of` loops.